include/cantera/thermo/SurfPhase.h

```diff
@@ -189,8 +210,8 @@ class SurfPhase : public ThermoPhase
      *
      * @param k Optional parameter indicating the species. The default
      *          is to assume this refers to species 0.
-     * @return
-     *   Returns the standard Concentration in units of m3 kmol-1.
+     * @return the standard concentration in units of m^2/kmol for surface phases or
```

Comment:
I don't understand the units in these docstrings.  Aren't they inverted from those of concentrations?

Reply:
Good catch. I was just correcting the fact that it's m^2 for surfaces, but this is clearly inverted as well.